src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,15 @@
-export const TodoList = () => {};
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => (
+      <TodoInfo todo={todo} key={todo.id} />
```

Comment:
The `key` prop is correctly set to `todo.id`, which is important for React's reconciliation process. Make sure that each `todo` object has a unique and valid `id` to prevent any potential issues with component rendering.